### PR TITLE
Avoid installing Coveralls in Travis CI for Perl 5.10

### PR DESCRIPTION
**Pull request by nuno-agostinho**
This PR avoids installing Coveralls in Travis CI for the Perl 5.10 job.

~~Also, fixes the `ENV` column in GitHub when running a Travis CI job. Currently showing `One of the secure variables in your .travis.yml has an invalid format.`. Example: https://github.com/Ensembl/ensembl-vep/runs/26175837180~~

### CODE REVIEW

**Review by nakib103 — COMMENTED** [review]
Probably the failure is coming from Devel::Cover v1.44
https://github.com/pjcj/Devel--Cover/compare/v1.43...v1.44

File: .travis.yml [hunk 1]
```diff
@@ -8,12 +9,12 @@ perl:
   - "5.26"
 
 env:
-  matrix:
-  - COVERALLS=true  DB=mysql
+  jobs:
```
[the remark this answers is not in the document]
Comment (nuno-agostinho) [reply .travis.yml:12]: The Travis CI build validator was notifying that `matrix` is an alias for `jobs` at https://app.travis-ci.com/github/Ensembl/ensembl-vep/jobs/621783642/config:

Screenshot 2024-06-14 at 11 12 21

**Comment by nuno-agostinho** [comment]
Hi @nakib103, there's no point in installing Coveralls because it's not run for 5.10 anyway. We might as well just skip its installation.

**Review by nakib103 — APPROVED** [review]
Thanks @nuno-agostinho, the unit test are working!
Coveralls report generation failing which seems to be long standing issue and can be addressed in separate PR.

**Pull request merged**